ui/pages/confirm-transaction-base/gas-details-item/gas-details-item.js

```diff
@@ -0,0 +1,135 @@
+
+import Box from '../../../components/ui/box';
+import GasTiming from '../../../components/app/gas-timing/gas-timing.component';
+import I18nValue from '../../../components/ui/i18n-value';
+import InfoTooltip from '../../../components/ui/info-tooltip/info-tooltip';
+import LoadingHeartBeat from '../../../components/ui/loading-heartbeat';
+import TransactionDetailItem from '../../../components/app/transaction-detail-item/transaction-detail-item.component';
+import UserPreferencedCurrencyDisplay from '../../../components/app/user-preferenced-currency-display';
+
+const HeartBeat = () =>
+  process.env.IN_TEST === 'true' ? null : <LoadingHeartBeat />;
+
+const GasDetailItem = ({
+  hexMaximumTransactionFee,
+  hexMinimumTransactionFee,
+  isMainnet,
+  maxFeePerGas,
+  maxPriorityFeePerGas,
+  supportsEIP1559,
+  txData,
+  useNativeCurrencyAsPrimaryCurrency,
+}) => {
+  const t = useI18nContext();
+  return (
+    <TransactionDetailItem
+      key="gas-item"
+      detailTitle={
+        <Box display="flex">
+          <Box marginRight={1}>
+            <I18nValue messageKey="transactionDetailGasHeadingV2" />
+          </Box>
+          <span className="gas-details-item__estimate">
+            (<I18nValue messageKey="transactionDetailGasInfoV2" />)
+          </span>
+          <InfoTooltip
+            contentText={
+              <>
+                <p>
+                  {t('transactionDetailGasTooltipIntro', [
+                    isMainnet ? t('networkNameEthereum') : '',
+                  ])}
+                </p>
+                <p>{t('transactionDetailGasTooltipExplanation')}</p>
+                <p>
+                  <a
+                    href="https://community.metamask.io/t/what-is-gas-why-do-transactions-take-so-long/3172"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    {t('transactionDetailGasTooltipConversion')}
+                  </a>
+                </p>
+              </>
+            }
+            position="top"
+          />
+        </Box>
+      }
+      detailTitleColor={COLORS.BLACK}
+      detailText={
+        <div className="confirm-page-container-content__currency-container">
+          <HeartBeat />
+          <UserPreferencedCurrencyDisplay
+            type={SECONDARY}
+            value={hexMinimumTransactionFee}
+            hideLabel={Boolean(useNativeCurrencyAsPrimaryCurrency)}
+          />
+        </div>
+      }
+      detailTotal={
+        <div className="confirm-page-container-content__currency-container">
+          <HeartBeat />
+          <UserPreferencedCurrencyDisplay
+            type={PRIMARY}
+            value={hexMinimumTransactionFee}
+            hideLabel={!useNativeCurrencyAsPrimaryCurrency}
+          />
+        </div>
+      }
+      subText={t('editGasSubTextFee', [
+        <Box key="editGasSubTextFeeLabel" display="inline-flex">
+          <Box marginRight={1}>
+            <b>
```

Comment:
We try to use css to bold text, instead of `<b>`

Reply:
Yep fixed it.